kuksa_databroker/databroker/src/viss/v2/conversions.rs

```diff
@@ -0,0 +1,372 @@
+            }
+            broker::DataType::Int64 => i64::try_from(self).map(broker::DataValue::Int64),
+            broker::DataType::Uint8 | broker::DataType::Uint16 | broker::DataType::Uint32 => {
+                u32::try_from(self).map(broker::DataValue::Uint32)
+            }
+            broker::DataType::Uint64 => u64::try_from(self).map(broker::DataValue::Uint64),
+            broker::DataType::Float => f32::try_from(self).map(broker::DataValue::Float),
+            broker::DataType::Double => f64::try_from(self).map(broker::DataValue::Double),
+            // broker::DataType::Timestamp => {
+            //     SystemTime::try_from(self).map(broker::DataValue::Timestamp)
+            // }
+            broker::DataType::StringArray => {
+                Vec::<String>::try_from(self).map(broker::DataValue::StringArray)
+            }
+            broker::DataType::BoolArray => {
+                Vec::<bool>::try_from(self).map(broker::DataValue::BoolArray)
+            }
+            broker::DataType::Int8Array
+            | broker::DataType::Int16Array
+            | broker::DataType::Int32Array => {
+                Vec::<i32>::try_from(self).map(broker::DataValue::Int32Array)
+            }
+            broker::DataType::Int64Array => {
+                Vec::<i64>::try_from(self).map(broker::DataValue::Int64Array)
+            }
+            broker::DataType::Uint8Array
+            | broker::DataType::Uint16Array
+            | broker::DataType::Uint32Array => {
+                Vec::<u32>::try_from(self).map(broker::DataValue::Uint32Array)
+            }
+            broker::DataType::Uint64Array => {
+                Vec::<u64>::try_from(self).map(broker::DataValue::Uint64Array)
+            }
+            broker::DataType::FloatArray => {
+                Vec::<f32>::try_from(self).map(broker::DataValue::FloatArray)
+            }
+            broker::DataType::DoubleArray => {
+                Vec::<f64>::try_from(self).map(broker::DataValue::DoubleArray)
+            }
+            // broker::DataType::TimestampArray => {}
+            _ => Err(Error::ParseError),
+        }
+    }
+}
+
+impl std::fmt::Display for broker::DataType {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.pad(match self {
+            broker::DataType::String => "string",
+            broker::DataType::Bool => "bool",
+            broker::DataType::Int8 => "int8",
+            broker::DataType::Int16 => "int16",
+            broker::DataType::Int32 => "int32",
+            broker::DataType::Int64 => "int64",
+            broker::DataType::Uint8 => "uint8",
+            broker::DataType::Uint16 => "uint16",
+            broker::DataType::Uint32 => "uint32",
+            broker::DataType::Uint64 => "uint64",
+            broker::DataType::Float => "float",
+            broker::DataType::Double => "double",
+            broker::DataType::Timestamp => "timestamp",
```

Comment:
Not specifically related to this PR but timestamp/timestamp[] was removed from VSS some time ago, so we could possibly remove it as well from KUKSA

Reply:
I just removed them here. They were not supported by databroker anyway and would just throw an error if anyone tried to use them.